File: c++/PascalTriangle.cpp
#include<iostream>
using namespace std;

int fun(int n){
	int ctr=1;

	for(int i=2;i<=n;i++){
		ctr*=i;
	}

	return ctr;
}

int main(){

	int n=0;
	cin>>n;

	for(int i=0;i<n;i++){
		for(int j=0;j<=i;j++){
			cout<<(fun(i)/fun(j)+fun(i-j));
		}
		cout<<"\n";
	}

	return 0;
}

File: python/PascalTriangle.py
def fun(n):
	ctr=1

	for i in range(2,n+1):
		ctr*=i

	return ctr


n=4
for i in range(0,n):
	for i in range(0,i+1):
		print((fun(i)/fun(j)+fun(i-j)))

	print('\n')